form-sharing/1-getNewKoboForms.js

```diff
@@ -0,0 +1,10 @@
+getForms({}, state => {
+  const keywords = ['price', 'prix', 'bns', 'nrgt', 'grm', 'feedback'];
+
+  const checkForKeyWords = name => {
+    return keywords.some(keyword => name.toLowerCase().includes(keyword));
+  };
+
+  state.data = state.data.results.filter(form => checkForKeyWords(form.name));
```

Comment:
Since you need to use the kobo forms data in down stream jobs. I would recommend you add the data to state using a different key. Eg: `state.koboForms`, 

Don't use `state.data` to store information that you need in down stream operations because `state.data` is a temporary information store, usually used to save the result of particular operation.